Mathlib/Algebra/Order/Monoid/Defs.lean

```diff
@@ -123,29 +137,53 @@ set_option linter.deprecated false in
 #align bit0_pos bit0_pos
 
 /-- A linearly ordered additive commutative monoid. -/
-class LinearOrderedAddCommMonoid (α : Type*) extends LinearOrder α, OrderedAddCommMonoid α
+class LinearOrderedAddCommMonoid (α : Type*) extends OrderedAddCommMonoid α, LinearOrder α
 #align linear_ordered_add_comm_monoid LinearOrderedAddCommMonoid
 
+attribute [instance 100] LinearOrderedAddCommMonoid.toOrderedAddCommMonoid
+attribute [instance 50] LinearOrderedAddCommMonoid.toLinearOrder
+attribute [instance 0] LinearOrderedAddCommMonoid.toMin
+attribute [instance 0] LinearOrderedAddCommMonoid.toMax
+attribute [instance 0] LinearOrderedAddCommMonoid.toOrd
+
 /-- A linearly ordered commutative monoid. -/
 @[to_additive]
-class LinearOrderedCommMonoid (α : Type*) extends LinearOrder α, OrderedCommMonoid α
+class LinearOrderedCommMonoid (α : Type*) extends OrderedCommMonoid α, LinearOrder α
 #align linear_ordered_comm_monoid LinearOrderedCommMonoid
 
-attribute [to_additive existing] LinearOrderedCommMonoid.toOrderedCommMonoid
```

Comment:
Was this deleted intentionally?

Reply:
It's #660. We don't need to add this for the first projection. I think I should add this for the other projection.